src/main/kotlin/apply/application/ApplicantService.kt

```diff
@@ -70,6 +73,17 @@ class ApplicantService(
         }
     }
 
+    fun resetPassword(resetPasswordRequest: ResetPasswordRequest): String {
```

Comment:
코드가 길어지니 변수 이름으로 `request`도 적절해 보여요.

```suggestion
    fun resetPassword(request: ResetPasswordRequest): String {
```